resolvers/product/index.js

```diff
@@ -0,0 +1,55 @@
+const data = require("./productListing.json");
+
+const ProductListingSort = {
+    A_Z: "A_Z",
+    Z_A: "Z_A",
+    LOW_TO_HIGH: "LOW_TO_HIGH",
+    HIGH_TO_LOW: "HIGH_TO_LOW",
+    POPULAR: "POPULAR",
+    NEW: "NEW",
+    FAVOURITES: "FAVOURITES"
```

Comment:
We could remove these sort types, I think we don't have attributes on `Product` to identify them.